src/concepts/buying_storing_tft.md

```diff
@@ -24,6 +24,6 @@ Discover step-by-step instructions on buying and storing TFTs across different p
 - [Bettertoken (Farmers)](https://library.threefold.me/info/manual/#/tokens/threefold__tft_bettertoken)
 
 ## Other Related Manuals on TFT
-- [Store TFTs on Hardware Wallets](https://forum.threefold.io/t/storing-tft-on-hardware-wallets/802)
-- [Storing TFTs on TF Connect App](https://library.threefold.me/info/manual/#/tokens/threefold__wiki_3bot_connect_wallet)
-- [Bridge TFT between Stellar and Binance Smart Chain](https://library.threefold.me/info/manual/#/tokens/threefold__tft_bsc_bridge)
+- [Store TFTs on Hardware Wallets](../../TF_Farmer_Guide/TF_Complete_Farmer_Guide/farmer_guide.html?highlight=hardware#hardware-wallet)
+- [Storing TFTs on TF Connect App](../../TF_Farmer_Guide/TF_Complete_Farmer_Guide/farmer_guide.html#create-a-wallet)
+- [Bridge TFT between Stellar and Binance Smart Chain](../../TF_Farmer_Guide/TF_Complete_Farmer_Guide/farmer_guide.html?highlight=tfchain%20bridge#how-to-use-the-tfchain-stellar-bridge)
```

Comment:
if i use https for example, https://manual.grid.tf/TF_Farmer_Guide/TF_Complete_Farmer_Guide/farmer_guide.html#how-to-use-the-bsc-stellar-bridge instead of putting md file will it work? iwant to refer them to a specific # section of the md file, not the whole md file @mohamedamer453